src/frontend/komponenter/Felleskomponenter/Hendelsesoversikt/komponenter/HendelseItem.tsx

```diff
@@ -18,24 +24,47 @@ const Hendelsesbeskrivelse = styled(BodyShort)`
     padding: ${ASpacing2} 0;
 `;
 
-const ListeElement = styled.li`
-    padding: ${ASpacing3} ${ASpacing3} ${ASpacing3} ${ASpacing9};
+const StyledVStack = styled(VStack)`
+    margin-bottom: ${ASpacing6};
 `;
 
+const Sirkel = styled.div`
+    border: 2px solid ${ABorderDefault};
+    height: ${ASpacing4};
+    width: ${ASpacing4};
+    border-radius: 50%;
+`;
+
+const Strek = styled.div`
+    height: 100%;
+    width: 1px;
+    background: ${ABorderSubtle};
+    top: 2px;
+    left: 0.5rem;
+    transform: translateX(-0.5px);
+`;
 const HendelseItem = ({ hendelse }: IHendelseItemProps) => (
-    <ListeElement>
-        <BodyShort weight="semibold">{hendelse.tittel}</BodyShort>
-        {hendelse.beskrivelse && (
-            <Hendelsesbeskrivelse>{hendelse.beskrivelse}</Hendelsesbeskrivelse>
-        )}
-        <BodyShort textColor="subtle">{`${hendelse.dato}`}</BodyShort>
-        <BodyShort textColor="subtle">{`${hendelse.utførtAv} ${
-            hendelse.rolle.toString() !== BehandlerRolle[BehandlerRolle.SYSTEM] &&
-            behandlerRoller[hendelse.rolle]
-                ? `(${behandlerRoller[hendelse.rolle].navn})`
-                : ''
-        }`}</BodyShort>
-    </ListeElement>
+    <li>
+        <HStack gap="4">
+            <VStack align="center">
+                <Sirkel />
+                <Strek />
+            </VStack>
```

Comment:
Dette blir det samme som å bruke :before og :after med styling som posisjonerer elementene. Det er egentlig en HStack bestående av et ikon og dataen for historikkinnslaget, så har kodet det sånn så det er lettere å lese. Litt teit å måtte kode sirkel og strek på denne måten, men fant ikke noe å bruke fra aksel som passet, så jeg har beholdt det som før men med en del forenkling 😊
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/835064ca-9005-4df2-8a9b-e63d6915f243)

Reply:
Synes det ble veldig lesbart 👍